docker/ubuntu1604/Dockerfile

```diff
@@ -52,8 +55,9 @@ RUN echo "deb http://apt.openvstorage.com unstable main" > /etc/apt/sources.list
     	    libxio0 libxio0-dbg libxio0-dbgsym \
     	    libxio-dev libxio-dev-dbgsym && \
     apt-get install -y libev4 && \
-    cd /root && apt-get download --allow-unauthenticated arakoon && \
+    cd /opt/tmp_docker && apt-get download --allow-unauthenticated arakoon && \
```

Comment:
to check: is an apt-get install arakoon not possible these days instead of the download/install sequence hack?